integration-cli/docker_cli_rmi_test.go

```diff
@@ -14,26 +15,22 @@ func (s *DockerSuite) TestRmiWithContainerFails(c *check.C) {
 
 	// create a container
 	out, _, err := dockerCmdWithError("run", "-d", "busybox", "true")
-	if err != nil {
-		c.Fatalf("failed to create a container: %s, %v", out, err)
-	}
+	// failed to create a container
+	c.Assert(err, checker.IsNil, check.Commentf("out: %s", out))
```

Comment:
I wonder if we can use `dockerCmd` instead of `dockerCmdWithError` here, so that the assert for `err` is no need any more.